tests/scr-support_scrolled-continuous/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" lang="en">
+	<head>
```

Comment:
I copied this from another test, so I am not sure of the statement "all use 2-space indent". It may be true that all the tests I have made use tabs, actually... Let me look at this later for my tests in general...

(It reflects my editor setting, where tab is 4 spaces, and is all inherited from the days when I used to program in python...)